rancher/rancher
# bump the SUC version in the Dockerfile

## Issue: 

 
https://github.com/rancher/rancher/issues/41511

## Problem

 
The system-upgrade-controller chart needs to be updated to support k8s 1.26

## Solution


The support for 1.26 has been added to the chart system-upgrade-controller to 102.1.0+up0.5.0 ([PR](https://github.com/rancher/charts/pull/2630))

This PR is to bump the default system-upgrade-controller chart version used in Rancher so that  102.1.0+up0.5.0  will be used by default. 

## Testing


## Engineering Testing
### Manual Testing


The tests on the chart system-upgrade-controller have been done and described in the linked PR. 


### Automated Testing


n/a

## QA Testing Considerations

 
1. on freshly-installed Rancher v2.7-head, create a new RKE2/k3s cluster( with k8s 1.26 and any version less than 1.26)  , the chart  system-upgrade-controller 102.1.0+up0.5.0 should be installed 
2. upgrading Rancher from a previous version, the chart system-upgrade-controller should be upgraded to 102.1.0+up0.5.0  on any existing RKE2/K3s cluster automatically 


### Regressions Considerations


- cluster provisioning fails
- cluster updating falls
